dangerfile.js

```diff
@@ -9,6 +9,12 @@ if ( danger.github.pr.body.length < 10 ) {
 }
 
 // Use labels please!
-if ( danger.github.issue.labels.length === 0 ) {
-	warn( 'This PR is missing at least one label.' );
+const ghLabels = danger.github.issue.labels;
+if ( ! ghLabels.find( l => l.toLowerCase().includes( '[status]' ) ) ) {
+	warn( 'The PR is missing at least one [Status] label. Suggestions: `[Status] In Progress`, `[Status] Needs Review`' );
+}
+
+// Test instructions
+if ( ! danger.github.pr.body.includes( 'Testing instructions:' ) ) {
```

Comment:
The colon should probably be left out -- this PR for example has testing instructions, but no colon so is flagged

Reply:
Oh, I've added PR description _after_ these checks were added. To re-trigger these checks - CI job need to be restarted